sites/svelte.dev/.env.example

```diff
@@ -2,6 +2,10 @@
 GITHUB_CLIENT_ID=
 GITHUB_CLIENT_SECRET=
 
+# Path to local copy of Svelte relative from sites/svelte.dev. Used by the REPL.
+# Optional. The default value assumes the svelte repo and sites repo are in the same directory.
+LOCAL_SVELTE_PATH=../../../svelte
```

Comment:
maybe this shouldn't be enabled by default so that you don't have to have Svelte checked out and can just use the published version?
```suggestion
# LOCAL_SVELTE_PATH=../../../svelte
```

Reply:
The env var would only be used if you're explicitly testing with `?version=local`, so it would be fine if it isn't checked out. But I think it makes sense to comment it out too to denote that it's optional.